lite2/client.go

```diff
@@ -128,6 +150,7 @@ type Client struct {
 	trustingPeriod   time.Duration // see TrustOptions.Period
 	verificationMode mode
 	trustLevel       tmmath.Fraction
+	maxRetryAttempts int // see MaxRetryAttempts option
```

Comment:
should we start being more strict on int types? this may be better suited as an int32 or uint32 possibly, maybe even smaller.

Reply:
good idea